controllers/dns_helper.go

```diff
@@ -134,7 +121,7 @@ func (dh *dnsHelper) setEndpoints(mcgTarget *multicluster.GatewayTarget, dnsReco
 	}
 
 	sort.Slice(endpoints, func(i, j int) bool {
-		return endpoints[i].SetID() < endpoints[j].SetID()
+		return endpoints[i].Key().SetIdentifier < endpoints[j].Key().SetIdentifier
```

Comment:
```suggestion
		return endpoints[i].Key() < endpoints[j].Key()
```

Reply:
I can't do this, I get an error, that's why I wondered if you meant that I should use the setIdentifier here.